### PR TITLE
WIP disable livefyre/enable GH issues

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/docfx.json [hunk 1]
```diff
@@ -42,7 +42,11 @@
       "_navRel": "/foo",
       "searchScope": [
         "ASP.NET Core"
-      ]
+      ],
+      "ms.topic": "article",
```
Comment (mairaw) [inline aspnetcore/docfx.json]: you should now change to conceptual if you wanna add this in the global settings 😄

**Comment by Rick-Anderson** [comment]
Closing until we're ready.

File: aspnetcore/docfx.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
       "ms.topic": "conceptual",
       "feedback_system": "GitHub",
       "feedback_github_repo": "aspnet/docs",
-      "feedback_product_url": "https://developercommunity.visualstudio.com/spaces/61/index.html" 
+      "feedback_product_url": "https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md"
```
Comment (mairaw) [inline aspnetcore/docfx.json]: you'll need to configure this differently based on the feedback from Dan.

Comment (Rick-Anderson) [reply aspnetcore/docfx.json]: `@DamianEdwards` trumps @danroth27 
@danroth27  do you want your feedback of https://github.com/aspnet/home/issues or your bosses?

**Comment by scottaddie** [comment]
@Rick-Anderson It's my understanding that we'll have to force a publish when the *docfx.json* file changes. Do we know how to accomplish that in the OPS Portal?

**Comment by mairaw** [comment]
I know how to if you need help @scottaddie. But usually docfx.json changes automatically trigger one. Even the PR already properly shows the buttons on the pages.

**Pull request merged**